package.json

```diff
@@ -28,6 +28,7 @@
     "prop-types": "^15.7.2",
     "react": "^16.13.1",
     "react-app-rewired": "^2.1.6",
+    "react-date-range": "^1.0.3",
```

Comment:
jw - why is this added?

Reply:
Err I messed up on the dependencies from dateFilter lol : ((